src/Console/StdOut.php

```diff
@@ -6,13 +6,16 @@
  * Switches between ansi and non-ansi output
  */
 
-class StdOut
-{
+interface OutStream {
```

Comment:
Each interface or class needs its own file and needs to follow PSR conventions. So this class should be called `OutStreamInterface` and should be in a file called OutStreamInterface.php. The abstract class below should be called `AbstractFormattableStream` and be in a file called AbstractFormattableStream.php.

See https://www.php-fig.org/bylaws/psr-naming-conventions/